### PR TITLE
Toggle DisableFrameThrottling setting by clicking on FPS display

### PR DESCRIPTION
**Pull request by darko1979**
I made a suggestion few days ago about adding a toggle button for enabling/disabling frame throttling.
At first I thought about adding special button but it was easier to do it by clicking on FPS counter.
Not sure if I did it right, it was not working until I updated the setting inside presentation engine and updated the swapchain.
Tested on my phone and it seems to be working.

### CODE REVIEW

**Comment by PixelyIon** [comment]
@lynxnb You think it's a good idea to implement this in the meantime as we don't currently have dynamic settings with callbacks?

**Comment by lynxnb** [comment]
Yes I believe this could be useful in the meantime, but implementation need some work. I'll follow up with a review explaining what needs to be done (we already have a mechanism for runtime settings modification and reload).

**Review by lynxnb — CHANGES_REQUESTED** [review]
The `toggleDisableFrameThrottling` function should be removed in favor of adding a callback [here](https://github.com/skyline-emu/skyline/blob/ftx1/app/src/main/cpp/skyline/gpu/presentation_engine.cpp#L35) like this:
```cpp
state.settings->disableFrameThrottling.AddCallback(...);
```
Here is [AddCallback](https://github.com/skyline-emu/skyline/blob/ftx1/app/src/main/cpp/skyline/common/settings.h#L54). 

On the Kotlin side, code is not ready yet to handle updating new settings at runtime. You'd ideally just have to set the new value to SharedPreferences, then the system would automatically update settings. We'd like to have that system in place before merging this in the current state.

File: app/src/main/java/emu/skyline/EmulationActivity.kt [hunk 1]
```diff
@@ -251,6 +256,13 @@ class EmulationActivity : AppCompatActivity(), SurfaceHolder.Callback, View.OnTo
                         postDelayed(this, 250)
                     }
                 }, 250)
+                setOnClickListener {
+                    var result = toggleDisableFrameThrottling()
```
[the remark this answers is not in the document]
Comment (darko1979) [reply app/src/main/java/emu/skyline/EmulationActivity.kt]: Done, hope this is what you had in mind.

**Comment by bylaws** [comment]
this doesn't seem to work for me

**Comment by darko1979** [comment]
> this doesn't seem to work for me

Original code from first commit worked for me. With requested changes only color of the FPS indicator is changing on touch because settings are not propagated to the presentation layer.

**Comment by lynxnb** [comment]
It's expected for stuff to not work as of now. I'll soon add a commit to this PR wiring settings up so that they're correctly handed down to the native side. After that, things should work properly.

**Comment by lynxnb** [comment]
I haven't forgotten about this, I'll add the missing piece in the following days.

**Comment by lynxnb** [comment]
Done, also rebased while at it.

**Review by lynxnb — APPROVED** [review]

**Pull request merged**